.github/workflows/kitchen.yml

```diff
@@ -137,19 +92,13 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest]
-        ruby: ["3.1"] # macos-11.0 is not public for now
+        ruby: ['3.1']
```

Comment:
The `matrix` key will add the label for this run, it does not do any ruby setup hence keeping it. (The existing comment was irrelavent also not true any more, hence removed)